### PR TITLE
convert donald to a monorepo

### PR DESCRIPTION
**Pull request by titus-mumo**

### CODE REVIEW

File: packages/donald-backend/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "donald-backend",
```
Comment (ngareleo) [inline packages/donald-backend/package.json:2]: Let's call this package `hercules` I like the name😅

**Comment by ngareleo** [comment]
Let's try to test this by creating another package in donald that imports something from `hercules` to make sure it really works

File: packages/donald-backend/package.json (hunk 1 above)
Comment (titus-mumo) [reply packages/donald-backend/package.json:2]: Kinda like it too😅 I'll change it

File: .prettierignore [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
-supabase
-temp
-.dockerfile
-.env
-.gitignore
+packages/donald-backend/supabase
+packages/donald-backend/temp
+packages/donald-backend/.dockerfile
+packages/donald-backend/.env
+packages/donald-backend/.gitignore
```
Comment (ngareleo) [inline .prettierignore:9]: No need for these prefixes, for directories, just add a slash at the end like `supbase/`
Git will do a deep search

**Review by ngareleo — CHANGES_REQUESTED** [review]

**Pull request merged**